ExportData.cs

```diff
@@ -20,39 +20,27 @@ namespace Tracker
 {
     public static class ExportData
     {
-        private static string user;
-        private static string computerName;
-        private static string geo_x;
-        private static string geo_y;
-        private static string dynamoVersion;
-        private static string revitVersion;
-        private static string ip;
-        private static string filename;
+        internal static string user;
+        internal static string computerName;
+        internal static string dynamoVersion;
+        internal static string revitVersion;
+        internal static string ip;
+        internal static string latlng;
+        internal static string city;
+        internal static string country;
+        internal static string filename;
         private static string date;
 
-        internal static IList<IList<object>> Export(string filename, string dynamoversion)
+        internal static IList<IList<object>> Export()
         {
             IList<IList<object>> export = new List<IList<object>>();
 
-            user = Environment.MachineName;
-            computerName = Environment.UserName;
+            // Set the date
+            // @todo "hh" is incorrectly returning values in the afternoon. It should be "14" not "2" etc for any hour after midday
             date = DateTime.Now.ToString("yyyy-MM-dd hh:mm:ss");
```

Comment:
I;'d suggest to use the [ISO8601 standard format](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#the-round-trip-o-o-format-specifier) instead of custom formats.

Reply:
New issue created: https://github.com/teamtreedyn/Binoculars/issues/17 Therefore I'm marking this resolved.